include/pros/adi.hpp

```diff
@@ -222,9 +222,17 @@ class ADIAnalogIn : private ADIPort {
 	 * nearly 5 V
 	 */
 	using ADIPort::get_value;
+
+	std::uint8_t get_adi_port() const;
+
+	std::uint8_t get_smart_port() const;
+	
+	void set_adi_port(std::uint8_t adi_port);
+
+	void set_smart_port(std::uint8_t smart_port);
```

Comment:
I understand you need these for accessing private members in derived classes, but is it possible to make these private? I don't think we should add these to the public API.

Reply:
Also see below about get_adi_port and get_smart_port